tests/e2e/lib/pages/postFrontend.js

```diff
@@ -6,7 +6,7 @@ import { waitAndClick, waitForSelector } from '../page-helper';
 
 export default class PostFrontendPage extends Page {
 	constructor( page ) {
-		const expectedSelector = '#main article.post';
+		const expectedSelector = '.post';
```

Comment:
I reviewed all of the Core default themes, there was no consistent selector with the level of specificity of `#main article.post`: sometimes `#main` existed, sometimes it didn't. Sometimes it was a `<div>`, sometimes a `<main>`, sometimes it had `role="main"` defined. Sometimes it used `<article>` sometimes a `<div>`. However, they all use the `.post` class, given how long this class has been defined, I would expect it to continue into the future.

I don't think the lower specificity is a problem, since it's always loading a single post page: if there's no post, the `.post` class won't be used anywhere.

Reply:
Thanks!